src/main/java/org/json/XMLParserConfiguration.java

```diff
@@ -56,6 +59,11 @@ of this software and associated documentation files (the "Software"), to deal
      */
     private boolean convertNilAttributeToNull;
 
+    /**
+     * This will allow type conversion for values in XML if xsi:type attribute is defined
+     */
+    public Map<String, XMLXsiTypeConverter<?>> xsiTypeMap;
```

Comment:
Sorry, I finally had time for a closer look at the configuration changes. With the new changes in #543, this should be private with a Getter and no Setter. Preferably the getter should return an "[Unmodifiable Map](https://docs.oracle.com/javase/6/docs/api/java/util/Collections.html#unmodifiableMap(java.util.Map))". To prevent wrapping the collection every time the getter is called, storing the unmodifiable map in this variable would be acceptable.

Note in the Getter javadoc that the map is "unmodifiable"